packages/backend/src/graphql/schema.graphql

```diff
@@ -277,320 +299,329 @@ type Mutation {
 }
 
 type PageInfo {
-  "Paginating forwards: the cursor to continue."
+  "The cursor used to fetch the next page when paginating forwards."
   endCursor: String
-  "Paginating forwards: Are there more pages?"
+  "Indicates if there are more pages when paginating forwards."
   hasNextPage: Boolean!
-  "Paginating backwards: Are there more pages?"
+  "Indicates if there are more pages when paginating backwards."
   hasPreviousPage: Boolean!
-  "Paginating backwards: the cursor to continue."
+  "The cursor used to fetch the next page when paginating backwards."
   startCursor: String
 }
 
 type AssetsConnection {
+  "Information to aid in pagination."
   pageInfo: PageInfo!
+  "A list of edges representing assets and cursors for pagination."
   edges: [AssetEdge!]!
 }
 
 type AssetEdge {
+  "An asset node in the list."
   node: Asset!
+  "A cursor for paginating through the assets."
   cursor: String!
 }
 
 input ApproveIncomingPaymentInput {
-  "Unique identifier of the incoming payment to be approved. Note: Incoming Payment must be PENDING."
+  "Unique identifier of the incoming payment to be approved. Note: incoming payment must be PENDING."
   id: ID!
 }
 
 input CancelIncomingPaymentInput {
-  "Unique identifier of the incoming payment to be cancelled. Note: Incoming Payment must be PENDING."
+  "Unique identifier of the incoming payment to be canceled. Note: incoming payment must be PENDING."
   id: ID!
 }
 
 input CreateAssetInput {
-  "[ISO 4217 currency code](https://en.wikipedia.org/wiki/ISO_4217), e.g. `USD`"
+  "[ISO 4217 currency code](https://en.wikipedia.org/wiki/ISO_4217), e.g. `USD`."
```

Comment:
Maybe we can link to the docs for the assets page on this one? technically, it doesn't have to be the ISO 4217 currency code -> an ASE can support any kind of asset it may wish